window.go

```diff
@@ -351,7 +361,19 @@ func (w *Window) SetSize(width, height int) error {
 func (w *Window) GetFramebufferSize() (width, height int, err error) {
 	var wi, h C.int
 	C.glfwGetFramebufferSize(w.data, &wi, &h)
-	return int(wi), int(h), fetchError()
+	return int(wi), int(h), fetchError() // FIXME: width and height are allowed to be NULL in C API
```

Comment:
Can we fix this issue before merging the PR?

What is the problem exactly, and how should we best handle it?